cmd/hook.go

```diff
@@ -102,14 +104,22 @@ func runHookPreReceive(c *cli.Context) error {
 		oldCommitID := string(fields[0])
 		newCommitID := string(fields[1])
 		refFullName := string(fields[2])
+		total++
+		lastline++
 
 		// If the ref is a branch, check if it's protected
 		if strings.HasPrefix(refFullName, git.BranchPrefix) {
 			oldCommitIDs[count] = oldCommitID
 			newCommitIDs[count] = newCommitID
 			refFullNames[count] = refFullName
 			count++
+			fmt.Fprintf(os.Stdout, "*")
+			os.Stdout.Sync()
+
 			if count >= hookBatchSize {
+				fmt.Fprintf(os.Stdout, " Checking %d branches\n", count)
```

Comment:
I was about to suggest exactly this to avoid the timeouts! 😄

Reply:
Anyway, this message will print after a line of stars and dots, is that what you intended?
```
****...*.*.***.****.*.***.****.***.*.*.*..*** Checking 30 branches
```

Reply:
Yup. that's intended

Reply:
if you count the preceding `*` in that line and the lines above there are 30 of them. The * indicates branches

Reply:
Is it possible to disable these remote-messages?

Reply:
Please fire a new issue.